lib/tasks/create_coupons.rake

```diff
@@ -0,0 +1,7 @@
+desc "it creates coupon code for all users"
```

Comment:
put both the rake task in namespace named `coupons`
rake coupons:create
rake coupons:send

Reply:
Done.